TIGLViewer/src/modificators/ModificatorSectionsWidget.h

```diff
@@ -42,11 +77,11 @@ public slots:
     explicit ModificatorSectionsWidget(QWidget* parent = nullptr);
     ~ModificatorSectionsWidget();
 
-    void setCreateConnectedElementI(tigl::CreateConnectedElementI& elementI);
+    void setCreateConnectedElement(Ui::ElementModificatorInterface& element);
```

Comment:
Can't the argument be `const`?

```suggestion
    void setCreateConnectedElement(Ui::ElementModificatorInterface const& element);
```